piwik-sdk/src/main/java/org/piwik/sdk/Piwik.java

```diff
@@ -68,6 +68,16 @@ public synchronized Tracker newTracker(@NonNull String trackerUrl, int siteId) t
     }
 
     /**
+     * Configure Piwik to receive more than 5 custom variable
+     * http://piwik.org/faq/how-to/faq_17931/
+     * Changing this limit will affect all Tracker instances
+     * @param maxVariables
+     */
+    public void setMaxCustomVariables(int maxVariables){
+        CustomVariables.MAX_VARIABLES = maxVariables;
```

Comment:
I agree, that sounds like a clean solution with some javadoc explanation.
Am 24.03.2016 00:39 schrieb "Alexey Subbotin" <notifications@github.com>:

> In piwik-sdk/src/main/java/org/piwik/sdk/Piwik.java
> <https://github.com/piwik/piwik-sdk-android/pull/90#discussion_r57255556>:
>
> > @@ -68,6 +68,16 @@ public synchronized Tracker newTracker(@NonNull String trackerUrl, int siteId) t
> >      }
> >
> >      /**
> > +     * Configure Piwik to receive more than 5 custom variable
> > +     * http://piwik.org/faq/how-to/faq_17931/
> > +     * Changing this limit will affect all Tracker instances
> > +     * @param maxVariables
> > +     */
> > +    public void setMaxCustomVariables(int maxVariables){
> > +        CustomVariables.MAX_VARIABLES = maxVariables;
>
> I think the most elegant way will be to remove limit of custom vars
> completely.
> We just can assume that developers are responsible enough and read the doc
> strings.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/piwik/piwik-sdk-android/pull/90/files/9c1561bcf2ec4d190e9515b9fa198607815d8669#r57255556>
>